style.css

```diff
@@ -2,3 +2,184 @@
   padding: 0;
   margin: 0;
 }
+body{
+  color: #797979;
+  font-family: 'Source Sans Pro', sans-serif;
+}
+a{
+  font-family: "Lora",serif;
+  color: #555555;
+}
+footer {
+  width: 100%;
+  height: 600px;
+  background-color: #000000;
+}
+footer h6{
+  color: #ffffff;
+  font-size: 18px;
+  margin-top: 70px;
+  
+}
+
+footer p{
+  color: #797979;
+  font-size: 17px;
+  
+}
+
+.footer-input-btn{
+  background-color: #222;
+  color: #ff9907;
+  height: 30px;
+  width: 15%;
+  margin-left: 0;
+}
+
+.main-footer{
+  display: flex;
+  justify-content: space-between;   
+}
+.footer-content{
+  width: 25%;
+  margin: 30px; 
+}
+.footer-copyright{
+  margin-left: 400px;
+}
+
+
+.fimg{
+  width: 50px;
+  height: 50px;
+  padding: 2px;
+}
+.firstc{
+  margin-left:70px ;
+}
+.footer-input{
+  width: 75%;
+  height: 30px;
+}
+
+.container {
+  position: relative;
+  border: 3px solid orange;
+  height: 40px;
+  overflow: hidden;
+  margin-bottom: 10px;
+  width: 300px;
+}
+
+.text_input {
+  height: 40px;
+  width: 80%;
+  padding: 0;
+  line-height: 30px;
+  font-size: 20px;
+  padding: 0;
+  margin: 0px;
+  margin-left: 0px;
+  border: none;
+}
+.text_input:focus {
+  outline: none;
+}
+
+.btn {
+  position: absolute; 
+  height: 40px;
+  line-height: 40px;
+  right: 0;
+  top: 0;
+  margin: 0;
+  padding: 0;
+  background: orange;
+  color: black;
+  border: none;
+  width: 20%;
+  font-weight: bold;
+}
+
+.fa-angle-double-right{
+    font-size: 25px;
+}
+.footer-social-icons i{
+  font-size: 20px;
+  margin-right: 20px;
+  color: #797979;
+}
+.footer-social-icons i:hover{
+  
+  color: #ff9907;
+}
+
+.footer-text i,.footer-text a{
+  color: #ff9907;
+  text-decoration: none;
+}
+@media (min-width: 481px) and (max-width: 767px) {
+  .footer-content{
+    width: 100%;
+    margin: 15px; 
+}
+.main-footer{
+    display: flex;
+    justify-content: space-between; 
+    flex-wrap: wrap;  
+    margin: 10px;
+}
+footer {
+    width: 100%;
+    height: 1000px;
+    
+}
+.footer-copyright{
+  margin-left: 100px;
+}
+.fimg{
+  width: 80px;
+  height: 80px;
+  padding: 20px;
+}
+
+.container {
+  
+  width: 500px;
+}
+.footer-copyright{
+  margin-left: 20px;
+  margin-top: 10px;
+  text-align: center;
+}
+
+  
+  
+}
+
+@media (min-width: 768px) and (max-width: 1024px) {
+
+  .footer-content{
+      width: 45%;
+      margin: 15px; 
+  }
+  .main-footer{
+      display: flex;
+      justify-content: space-between; 
+      flex-wrap: wrap;  
+      margin: 10px;
+  }
+  footer {
+      width: 100%;
```

Comment:
there is noo need to repeat this property here, since you gave it to the footer and you don't want to change it 
```css
footer {
    width: 100%;
    height: 1000px;

}
```

Reply:
I have modified it